### PR TITLE
Add scrollpane to storybook

### PR DESCRIPTION
**Pull request by tjamesmac**
### What is the context of this PR?

Adds the scroll pane component to Storybook

Run into a few issues with the `.mdx` format whilst working on this ticket. Working with storybook is pretty cool though.

Getting a `.mdx` syntax highlighter is a really good idea, otherwise it gets hard to identify the `jsx` in the code.

Not sure if it's just my set up, but `prettier` doesn't work for me with `.mdx`. It apparently does support the file type but errors in the VScode terminal.

There is an issue with `.mdx` right now where viewing the source code in storybook prints out decorators and other bits. At the moment, the `Template.bind({})` doesn't work. I've left the decorator as is, even though it outputs more to the source.

I've made it so the text of the scroll pane is editable within storybook itself.

I've also added a few divs to style the scroll pane so it doesn't look crazy

Also also renamed the ScrollPane from StyledScrollPane because that was the name of the component in the Storybook source code.

| Weird sb bug  |  
| ------------- | 
|Screenshot 2020-09-23 at 14 47 14| 

Issue being discussed here
https://github.com/storybookjs/storybook/issues/11542

### How to review 

1. Run `yarn storybook`
2. Check out the scrollpane component
3. I've removed actions from the stories (it shouldn't appear in the tab next to controls)
4. Review the documentation under the `Docs` tab
5. Check the source code under `Show code` in the docs tab

### CODE REVIEW

**Review by zimboflyman — APPROVED** [review]
Looks good and especially impressed you added research links and tips on using scrollbars!
shot!

File: eq-author/src/storybook/Components/ScrollPane.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+      </div>
+    ),
+  ]}
+/>
+
+# Scroll Pane
+
+Hides overflowing content and makes it scrollable.
+
+There is an optional property that the component accepts `scrollToTop`
```
Comment (MattElcock) [inline eq-author/src/storybook/Components/ScrollPane.stories.mdx:31]:
```suggestion
There is an optional property that the component accepts `scrollToTop`.
```

File: eq-author/src/storybook/Components/ScrollPane.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+Use this pattern sparingly. We have tried to limit the amount of scroll panes present within the Author application.
+
+Having multiple scroll panes can frustrate and confuse users (Mouse wheel behaviour).
+
+Note: The black border surrounding the scroll pane is for example purposes only. It's used to highlight where the content begins scrolling.
+
+## When not to use this component
+
+Do not use this component if there are other ways to obscure content from the user
```
Comment (MattElcock) [inline eq-author/src/storybook/Components/ScrollPane.stories.mdx:57]:
```suggestion
Do not use this component if there are other ways to obscure content from the user.
```

File: eq-author/src/storybook/Components/ScrollPane.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+  >
+    {(args) => <ScrollPane {...args} />}
+  </Story>
+</Canvas>
+
+## When to use this component
+
+Use this pattern sparingly. We have tried to limit the amount of scroll panes present within the Author application.
+
+Having multiple scroll panes can frustrate and confuse users (Mouse wheel behaviour).
```
Comment (MattElcock) [inline eq-author/src/storybook/Components/ScrollPane.stories.mdx:51]:
```suggestion
Having multiple scroll panes can frustrate and confuse users (mouse wheel behaviour).
```

**Review by MattElcock — APPROVED** [review]
Looks really good, mate. I've popped a few very minor things; just a few full stops and a rogue capital letter. But, it can go in without these if you want - so I've approved it anyway.

**Review by zimboflyman — APPROVED** [review]

**Review by MattElcock — APPROVED** [review]

**Pull request merged**